scripts/Docker/readme.md

```diff
@@ -0,0 +1,76 @@
+# Librealsense Docker
+
```

Comment:
Add opening para : 
- intro, 
- target audience
-  the  expected outcome of following the guide

Reply:
Done